app/routes/datasets/downloads.py

```diff
@@ -27,6 +30,46 @@
 
 router: APIRouter = APIRouter()
 
+OPENAPI_EXTRA_AOI = {
```

Comment:
For the datamart route, I put this in its `__init__.py` to clean things up a bit.